Documentation/Contributing/development-flow.md

```diff
@@ -287,7 +287,7 @@ Signed-off-by: First Name Last Name <email address>
 
 The `component` **MUST** be in the [list checked by the CI](https://github.com/rook/rook/blob/master/.commitlintrc.json).
 
-Note: sometimes you will feel like there is not so much to say, for instance if you are fixing a typo in a text.
+!!! note: sometimes you will feel like there is not so much to say, for instance if you are fixing a typo in a text.
```

Comment:
The `!!! note` must be on its own line to be rendered correctly in the docs, with the content of the note indented. You can search for other places in the docs for consistent formatting.
```suggestion
!!! note
    Sometimes you will feel like there is not so much to say, for instance if you are fixing a typo in a text.
```

Reply:
I have applied and mentioned changed and updated the PR.